deps.toml

```diff
@@ -84,6 +84,7 @@ findsecbugs-plugin = { module = "com.h3xstream.findsecbugs:findsecbugs-plugin",
 spotbugs-annotations = { module = "com.github.spotbugs:spotbugs-annotations", version = "4.6.0" }
 otel-bom = {module = "io.opentelemetry:opentelemetry-bom", version = "1.14.0"}
 otel-semconv = {module = "io.opentelemetry:opentelemetry-semconv", version = "1.14.0-alpha"}
+otel-sdk = {module = "io.opentelemetry:opentelemetry-sdk-metrics", version = "1.15.0"}
```

Comment:
should this be 1.14.0 to match the otel-bom?

Reply:
Updated.